src/lovok_handle_internal.cpp

```diff
@@ -53,52 +53,58 @@ LovokStatusCode ParseHeader(FileWrapper *fileWrapper, Box *header) {
     return VALID_FILE;
 }
 
-bool AddOverflow(uint64_t a, uint64_t b, uint64_t *result) {
+bool OverflowCheck(uint64_t a, uint64_t b) {
```

Comment:
I realize you split these in two, but I think the word "add" still needs to be here for clarity.

Reply:
I updated this method to `AddOverflowCheck`. I also updated `UnderflowCheck` to be `SubtractUnderflowCheck`.